<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow {background-image: url(:/pics/тест.png);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" rowspan="2" colspan="7">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>200</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="6">
       <widget class="QLineEdit" name="K3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
background-color: rgba(25, 25, 112,0.5);
color: rgb(254, 254, 17);
border: 0px;
border-radius: 0px;
	border-color: rgb(255, 255, 255,0.3);
	selection-color: rgb(248, 188, 255,0.8);
	selection-background-color: rgb(170, 85, 255,0.8);

}
</string>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>50</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_4">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Poor Richard</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel { background-color: rgba(25, 25, 112);
color: rgb(244, 164, 96); }</string>
        </property>
        <property name="text">
         <string>A</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="K1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
background-color: rgba(25, 25, 112,0.5);
color: rgb(254, 254, 17);
border: 0px;
border-radius: 0px;
	border-color: rgb(255, 255, 255,0.3);
	selection-color: rgb(248, 188, 255,0.8);
	selection-background-color: rgb(170, 85, 255,0.8);

}
</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="maxLength">
         <number>32767</number>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLabel" name="label_3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Poor Richard</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel { background-color: rgba(25, 25, 112);
color: rgb(244, 164, 96); }</string>
        </property>
        <property name="text">
         <string>C</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Poor Richard</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel { background-color: rgba(25, 25, 112);
color: rgb(244, 164, 96); }</string>
        </property>
        <property name="text">
         <string>B</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>50</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="K2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
background-color: rgba(25, 25, 112,0.5);
color: rgb(254, 254, 17);
border: 0px;
border-radius: 0px;
	border-color: rgb(255, 255, 255,0.3);
	selection-color: rgb(248, 188, 255,0.8);
	selection-background-color: rgb(170, 85, 255,0.8);

}
</string>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" rowspan="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="1">
       <widget class="QPushButton" name="btn5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: rgba(25, 25, 112,0.8);
color: rgb(255, 165, 0);
border: 2px;
border-radius: 2px;
}

QPushButton:hover {
background-color: rgba(106, 90, 205, 0.4);
	border-color: rgb(255, 203, 12);
}

QPushButton:pressed {
background-color: rgba(72, 61, 139, 0.6);
}

QPushButton:disabled {
background-color: rgba(0, 0, 0, 0.2);
}</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btn8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: rgba(25, 25, 112,0.8);
color: rgb(255, 165, 0);
border: 2px;
border-radius: 2px;
}

QPushButton:hover {
background-color: rgba(106, 90, 205, 0.4);
	border-color: rgb(255, 203, 12);
}

QPushButton:pressed {
background-color: rgba(72, 61, 139, 0.6);
}

QPushButton:disabled {
background-color: rgba(0, 0, 0, 0.2);
}</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="btn2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: rgba(25, 25, 112,0.8);
color: rgb(255, 165, 0);
border: 2px;
border-radius: 2px;
}

QPushButton:hover {
background-color: rgba(106, 90, 205, 0.4);
	border-color: rgb(255, 203, 12);
}

QPushButton:pressed {
background-color: rgba(72, 61, 139, 0.6);
}

QPushButton:disabled {
background-color: rgba(0, 0, 0, 0.2);
}</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnC">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: rgba(25, 25, 112,0.8);
color: rgb(255, 165, 0);
border: 2px;
border-radius: 2px;
}

QPushButton:hover {
background-color: rgba(106, 90, 205, 0.4);
	border-color: rgb(255, 203, 12);
}

QPushButton:pressed {
background-color: rgba(72, 61, 139, 0.6);
}

QPushButton:disabled {
background-color: rgba(0, 0, 0, 0.2);
}</string>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btn9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: rgba(25, 25, 112,0.8);
color: rgb(255, 165, 0);
border: 2px;
border-radius: 2px;
}

QPushButton:hover {
background-color: rgba(106, 90, 205, 0.4);
	border-color: rgb(255, 203, 12);
}

QPushButton:pressed {
background-color: rgba(72, 61, 139, 0.6);
}

QPushButton:disabled {
background-color: rgba(0, 0, 0, 0.2);
}</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="start_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: rgba(25, 25, 112,0.8);
color: rgb(255, 165, 0);
border: 2px;
border-radius: 2px;
}

QPushButton:hover {
background-color: rgba(106, 90, 205, 0.4);
	border-color: rgb(255, 203, 12);
}

QPushButton:pressed {
background-color: rgba(72, 61, 139, 0.6);
}

QPushButton:disabled {
background-color: rgba(0, 0, 0, 0.2);
}</string>
        </property>
        <property name="text">
         <string>Рассчитать </string>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: rgba(25, 25, 112,0.8);
color: rgb(255, 165, 0);
border: 2px;
border-radius: 2px;
}

QPushButton:hover {
background-color: rgba(106, 90, 205, 0.4);
	border-color: rgb(255, 203, 12);
}

QPushButton:pressed {
background-color: rgba(72, 61, 139, 0.6);
}

QPushButton:disabled {
background-color: rgba(0, 0, 0, 0.2);
}</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="btn_point">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: rgba(25, 25, 112,0.8);
color: rgb(255, 165, 0);
border: 2px;
border-radius: 2px;
}

QPushButton:hover {
background-color: rgba(106, 90, 205, 0.4);
	border-color: rgb(255, 203, 12);
}

QPushButton:pressed {
background-color: rgba(72, 61, 139, 0.6);
}

QPushButton:disabled {
background-color: rgba(0, 0, 0, 0.2);
}</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
        <property name="shortcut">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="btn1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: rgba(25, 25, 112,0.8);
color: rgb(255, 165, 0);
border: 2px;
border-radius: 2px;
}

QPushButton:hover {
background-color: rgba(106, 90, 205, 0.4);
	border-color: rgb(255, 203, 12);
}

QPushButton:pressed {
background-color: rgba(72, 61, 139, 0.6);
}

QPushButton:disabled {
background-color: rgba(0, 0, 0, 0.2);
}</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btnAC">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: rgba(25, 25, 112,0.8);
color: rgb(255, 165, 0);
border: 2px;
border-radius: 2px;
}

QPushButton:hover {
background-color: rgba(106, 90, 205, 0.4);
	border-color: rgb(255, 203, 12);
}

QPushButton:pressed {
background-color: rgba(72, 61, 139, 0.6);
}

QPushButton:disabled {
background-color: rgba(0, 0, 0, 0.2);
}</string>
        </property>
        <property name="text">
         <string>AC</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="btn0">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: rgba(25, 25, 112,0.8);
color: rgb(255, 165, 0);
border: 2px;
border-radius: 2px;
}

QPushButton:hover {
background-color: rgba(106, 90, 205, 0.4);
	border-color: rgb(255, 203, 12);
}

QPushButton:pressed {
background-color: rgba(72, 61, 139, 0.6);
}

QPushButton:disabled {
background-color: rgba(0, 0, 0, 0.2);
}</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="btn6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: rgba(25, 25, 112,0.8);
color: rgb(255, 165, 0);
border: 2px;
border-radius: 2px;
}

QPushButton:hover {
background-color: rgba(106, 90, 205, 0.4);
	border-color: rgb(255, 203, 12);
}

QPushButton:pressed {
background-color: rgba(72, 61, 139, 0.6);
}

QPushButton:disabled {
background-color: rgba(0, 0, 0, 0.2);
}</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btn4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: rgba(25, 25, 112,0.8);
color: rgb(255, 165, 0);
border: 2px;
border-radius: 2px;
}

QPushButton:hover {
background-color: rgba(106, 90, 205, 0.4);
	border-color: rgb(255, 203, 12);
}

QPushButton:pressed {
background-color: rgba(72, 61, 139, 0.6);
}

QPushButton:disabled {
background-color: rgba(0, 0, 0, 0.2);
}</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="btn3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: rgba(25, 25, 112,0.8);
color: rgb(255, 165, 0);
border: 2px;
border-radius: 2px;
}

QPushButton:hover {
background-color: rgba(106, 90, 205, 0.4);
	border-color: rgb(255, 203, 12);
}

QPushButton:pressed {
background-color: rgba(72, 61, 139, 0.6);
}

QPushButton:disabled {
background-color: rgba(0, 0, 0, 0.2);
}</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_x1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>37</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>352</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="result_show_2">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"> Qlabel {
qproperty-alignment: 'AlignVCenter | AlignRight';
  border - bottom: 1px solid gray;
}

background-color : white;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
